### ISSUE TITLE
refactor/organize-texts-in-one-file

### ISSUE THREAD

**Issue by imranariffin** [hooligram/hooligram-client#30]
Currently all text files are defined within components. I don't think this is proper practice for localization. What do you think? @cglotr

**Comment by cglotr** [comment]
@imranariffin yeah, let's do localization since now the app is pretty small so it shouldn't impact the timeline too much. I don't have a lot of experience doing it tho. But, it should just be a key value mapping right?

**Comment by imranariffin** [comment]
Yeah, that's what I figured it should be. And the strings file should also
be flat I think.
